wcvendors/wcvendors
# Added: Log marketplace commissions to commissions table

### Changes proposed in this Pull Request:

Add support for logging all commissions within the marketplace. This includes both vendor and now the marketplace commissions. 

Closes #802 

### How to test the changes in this Pull Request:

1. Process an order with vendor products 
2. Admin commission split is logged for each product
3. Check commissions table to see orders now have commissions logged for both sides of the calculation. 

### Other information:
